pkg/utils/utils.go

```diff
@@ -278,26 +278,26 @@ func getNamespaceDefinition(namespace string) *corev1.Namespace {
 	}
 }
 
-func (u *utils) IsSNOCluster() (bool, error) {
+// GetPlatformType returns the cluster's platform type.
+// If it's not AWS, BareMetal, or None, it returns an unsupported platform error.
```

Comment:
I don't see where it returns "an unsupported platform error" (and it shouldn't anyways)